packages/location-field/src/stories/Desktop.story.tsx

```diff
@@ -252,34 +252,21 @@ export const WithUserSettings = (): JSX.Element => (
   />
 );
 
-export const WithOtherFirst = (): JSX.Element => {
-  const [otherControl, setOtherControl] = useState(false);
-  return (
-    <>
-      <label htmlFor="other-input">
-        renderOtherFirst?
-        <input
-          id="other-input"
-          type="checkbox"
-          checked={otherControl}
-          onChange={() => setOtherControl(!otherControl)}
-        />
-      </label>
-      <LocationField
-        currentPosition={currentPosition}
-        geocoderConfig={geocoderConfig}
-        getCurrentPosition={getCurrentPosition}
-        preferredLayers={["example_layer"]}
-        initialSearchResults={mockedGeocoderResponse.features}
-        inputPlaceholder="Select from location"
-        layerColorMap={layerColorMap}
-        locationType="from"
-        onLocationSelected={onLocationSelected}
-        renderOtherFirst={otherControl}
-        sortByDistance
-        style={{ fontFamily: "sans-serif" }}
-      />
-    </>
-  );
-};
-WithOtherFirst.parameters = a11yOverrideParameters;
+export const WithCustomResultsOrder = (): JSX.Element => (
+  <LocationField
+    currentPosition={currentPosition}
+    geocoderConfig={geocoderConfig}
+    getCurrentPosition={getCurrentPosition}
+    preferredLayers={["example_layer"]}
+    initialSearchResults={mockedGeocoderResponse.features}
+    inputPlaceholder="Select from location"
+    layerColorMap={layerColorMap}
+    locationType="from"
+    onLocationSelected={onLocationSelected}
+    geocoderResultsOrder={["OTHER", "STATIONS", "STOPS"]}
```

Comment:
Bit of a nit, but could we put the strings into an object so they're more consistent? Similar to how we do the view ids in otp-rr

Reply:
Can you clarify where in OTP-RR to look for this? Also, we're using a lot of array methods here, what's the benefit of making this an object?

Reply:
lib/actions/ui-constants.js! 

Really this should be an enum but we can't do that in JS so an object with keys that are strings is the cleanest way to do it. Using an enum is good practice, prevents silly mistakes, and prevents us from having to copy paste strings aroudn

Reply:
Added in [e2b352e](https://github.com/opentripplanner/otp-ui/pull/771/commits/e2b352e316bdd6501f4b5b49dc4f7f4be54f62c7)!